services/src/test/java/io/grpc/protobuf/services/ProtoReflectionServiceTest.java

```diff
@@ -114,6 +114,12 @@ public void tearDown() {
   }
 
   @Test
+  @SuppressWarnings("deprecation")
```

Comment:
Hint (no need to change): you can mark the method as `@Deprecated` instead of suppressing the warning.